### DISCUSSION TITLE
記事ページの先頭に「作成日」「作成者」「更新日」「更新者」の表示が欲しい

### DISCUSSION THREAD

**Discussion by satof3** [growilabs/growi#8733] [Ideas / Requests] [answered]
https://github.com/weseek/growi/issues/8630 より切り分けて議題化しています

# 現状
- 「作成日」「作成者」「更新日」「更新者」がフッターにのみ表示されている

# 今後の修正予定 (2024.4.17 時点)
- 更新履歴へのリンクをホバーすると作成日」「作成者」「更新日」「更新者」がページタイトルの位置からも確認できるようになる
- ![image](https://github.com/weseek/growi/assets/113958844/9d7622ea-a721-4383-90af-274b6645deae)

# 要望
- ホバーせずに表示させたい (v6 までのUI)
- 「題名」「タグ」「作成更新日」「作成更新者名」がまとまっていて欲しい
- ユーザーアイコンに特徴がなくても「作成更新者名」(「作成者」)がわかるようにしてほしい

**Comment by satof3** [comment]
GROWI 7.0.0 以降のデザイン方針としましては要素を減らし、レスポンシブへの対応をよりよくするためにv6以前にヘッダーにあった「作成更新日」「作成更新者名」を削除し、収納することで情報が整理され見やすいUIにする意図の元、画面設計を行いました。

https://github.com/weseek/growi/issues/8630 でのご意見を受け、デフォルト表示での変更は叶いませんが、設定によって表示できるようにすることを検討いたします。

つきましては詳細をお送りいただけるとより要望に沿ったデザインになると思います。

---
- 使う場面

  - PCより小さい画面でも必要性の有無(タブレット・モバイル端末)
  - 印刷やスクリーンショットなどの画面を表示させる以外の場面で「作成更新日」「作成更新者名」が必要になるか
- 優先順位
  -  「作成更新日」「作成更新者名」のどちらが必要性が高いか、またはどちらも同順位か
  -  現状 (v7.0.0 以降)で表示が不要もしくはサイズを減らしても良いと思う要素があるか
---

すべてのご要望を実現するのは難しいかもしれませんが、上記以外にもご意見お待ちしております。
また、具体的な表示位置のご希望やアイデアがありましたら是非ともお聞かせください。

**Comment by funakura** [comment]
ディスカッションへの移動ありがとうございます。

- 使う場面
   - PCより小さい画面でも必要性の有無(タブレット・モバイル端末)
  弊社のスタッフ系用途では、室内でのPCが主たる利用です。
  タブレットやスマホは、PCで参照して利用すると決めた特定記事をPCが使えない場所で見る場合の利用になます。
  そのため、作成日・作者の必要性はありません。
  
  - 印刷やスクリーンショットなどの画面を表示させる以外の場面で「作成更新日」「作成更新者名」が必要になるか
  印刷する時点で、記事を読むに値すると判断しているため付加情報は不要だと考えます

- 優先順位
  -  「作成更新日」「作成更新者名」のどちらが必要性が高いか、またはどちらも同順位か
    「作成更新日」が必要性が高いと考えます。（1年以内の記事でも、社内の特定イベントの後か前かなどの判断に使います）
  
  - 現状 (v7.0.0 以降)で表示が不要もしくはサイズを減らしても良いと思う要素があるか
  v7は全体的に要素のサイズが小さくなっているため、サイズが大きいと思う要素はあまりないです。
    - 「ページリスト」
    左側の階層から辿るためほとんど使いません。階層が上の記事だと表示しても量が多すぎて何が何やら。
    - 「コメント」
    クリックするとコメント欄に飛ぶと認識していますが、知りたいのはコメントの個数ですから「いいね」と同列で良いです。
    - 右下の「バージョン表示」
　    利用者に常時見える必要性は無いと思います。
    - 右下の「ショートカット」
　    左のヘルプアイコンの中で良いと思います。覚えてるメンバからすれば邪魔な表示です。

**Reply by satof3** [reply]
詳細なご意見・ご指摘ありがとうございました。
いただいた情報を元に改善や設定による表示を検討してまいります。

**Reply by funakura** [reply]
Ver.7.2.0  で、サイドバーに作成者、作成日、更新者、更新日の表示追加ありがとございます。

**Reply by satof3** [reply, marked as answer]
ご要望にお応えでき幸いです。
ご意見ありがとうございました。
